utils/download_utils.py

```diff
@@ -0,0 +1,82 @@
+import pandas as pd
+import numpy as np
+from shapely.geometry import Point
+from shapely.geometry.polygon import Polygon
+
+
+def parse_outline_data(raw_outline_data: str) -> np.array:
```

Comment:
It may be helpful to consider using [`numpy.typing.NDArray`](https://numpy.org/doc/stable/reference/typing.html#numpy.typing.NDArray) here for the return type hint. While I found the type hint readable as-is, numpy provides a typing component to specifically address np.array-like types. For example:
```python
import numpy.typing as npt

def parse_outline_data(raw_outline_data: str) -> npt.NDArray:
```

Reply:
Noted for the future! I should have removed this method anyways (parse_outline_data) as this method is no longer used.